test/test.ml

```diff
@@ -0,0 +1,53 @@
+open Letters
+
+let ( let* ) = Lwt.bind
+
+let get_ethereal_account_details () =
+  let open Yojson.Basic.Util in
+  (* see the README.md how to generate the account file and the path
+   * below is relative to the location of the executable under _build
+   *)
+  let json = Yojson.Basic.from_file "../../../ethereal_account.json" in
+  let username = json |> member "user" |> to_string in
```

Comment:
Is this a common pattern in OCaml to have intermediate `let` bindings? Wouldn't it have been easier to directly assign these to the record that you're returning? So
```ocaml
Lwt.return {
  sender = json |> member "user" |> to_string
  ...
```

Reply:
I'm not sure what is or is not common pattern in cases like this. Since I have that intermediary `smtp_node`, I thought this would be easier to read as it is more verbose.

Reply:
I think this is idiomatic because shorthand syntax can be used to create the records easily. @aronerben Could you resolve the comment if you agree so Miko can merge this PR?

Reply:
@jerben Ah yes, sorry, I thought it simply needed an approver to be able to merge it